segmentation/utils/Spatial_Analysis.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import math
+import skimage.measure
+from scipy.spatial.distance import cdist
+
+
+def calc_dist_matrix(label_map):
+    """Generate matrix of distances between center of pairs of cells
+        Args
+            label_map: numpy array with unique cells given unique pixel labels
+        Returns
+            dist_matrix: cells x cells matrix with the euclidian
+            distance between centers of corresponding cells"""
+
+    if len(np.unique(label_map)) < 3:
```

Comment:
This wasn't a very useful error check in the original function, and it fails if there's only two cells, so we can remove it and think of some better error checking later on